events/memberJoined.js

```diff
@@ -2,9 +2,9 @@ const moment = require('moment');
 const format = require('../momentFormat');
 
 module.exports = {
-  process: (bot, guild, member) => {
-    const userLogsChannel = guild.channels.find('name', 'user-logs');
-    const welcomeChannel = guild.channels.find('name', 'welcome-room');
+  process: (bot, member) => {
```

Comment:
I assume this has been tested? Just want to make sure this was just some stupid code from me and there wasn't a specific (undocumented...) reason we were passing in the guild here.